dev/ts_image.py

```diff
@@ -13,22 +12,32 @@
 """
 
 # Parse command line arguments
+import warnings
+import os
+
+from astropy.utils.exceptions import AstropyDeprecationWarning
+warnings.filterwarnings('ignore', category=AstropyDeprecationWarning)
```

Comment:
Agreed ... since we only support Astropy 1.0 (master at the moment) in Gammapy it should be possible to write code that doesn't raise any deprecation warnings.
@adonath If you get deprecation warnings from unrelated parts of Gammapy you don't have to fix them in this PR, but please don't silence them either.